akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/EffectfulActorContext.scala

```diff
@@ -91,9 +91,15 @@ import scala.reflect.ClassTag
     override def startTimerWithFixedDelay(key: Any, msg: T, delay: FiniteDuration): Unit =
       startTimer(key, msg, delay, Effect.TimerScheduled.FixedDelayMode)
 
+    override def startTimerWithFixedDelay(key: Any, msg: T, initialDelay: FiniteDuration, delay: FiniteDuration): Unit =
+      startTimer(key, msg, delay, Effect.TimerScheduled.FixedDelayModeWithInitialDelay(initialDelay))
+
     override def startTimerAtFixedRate(key: Any, msg: T, interval: FiniteDuration): Unit =
       startTimer(key, msg, interval, Effect.TimerScheduled.FixedRateMode)
 
+    override def startTimerAtFixedRate(key: Any, msg: T, initialDelay: FiniteDuration, interval: FiniteDuration): Unit =
+      startTimer(key, msg, interval, Effect.TimerScheduled.FixedRateModeWithInitialDelay(initialDelay))
```

Comment:
Effect.TimerScheduled.FixedRateMode doesn't include the `interval`, why should this include the `initialDelay`?

Also, maybe not important to distinguish between FixedRateMode and FixedRateModeWithInitialDelay in tests and we could just use FixedRateMode for both?

Reply:
Since `FixedRateMode` is an object and public to test code I could not add the initial delay without breaking existing tests.

Reply:
Since `FixedRateMode` is an object and public I could not just add the `initialDelay` without breaking existing code.

Reply:
yes, but my point was that why do we need to expose the initialDelay? It's not consistent with that interval isn't exposed

Reply:
I added it so `initialDelay` can also be specified when using the `ScheduleCommand`